system/logaccess/Grafana/CurlClient/GrafanaCurlClient.cpp

```diff
@@ -832,6 +873,24 @@ GrafanaLogAccessCurlClient::GrafanaLogAccessCurlClient(IPropertyTree & logAccess
             if (logMap.hasProp(logMapSearchColAtt))
                 m_podColumn.name = logMap.queryProp(logMapSearchColAtt);
         }
+        else if (streq(logMapType, "message"))
+        {
+            if (logMap.hasProp(logMapIndexPatternAtt))
```

Comment:
This code looks like it is duplicated multiple times.  A candidate for cleaning up in a later PR.
(Not being commoned up makes it look as though each of the different branches are doing something different..)